clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -965,14 +966,14 @@ void EmitAssemblyHelper::RunOptimizationPipeline(
     // configure the pipeline.
     OptimizationLevel Level = mapToLevel(CodeGenOpts);
 
-    if (LangOpts.SYCLIsDevice)
-      PB.registerPipelineStartEPCallback(
-          [&](ModulePassManager &MPM, OptimizationLevel Level) {
-            MPM.addPass(ESIMDVerifierPass(LangOpts.SYCLESIMDForceStatelessMem));
-            MPM.addPass(
-                SYCLPropagateAspectsUsagePass(/*ExcludeAspects=*/{"fp64"}));
-          });
-    else if (LangOpts.SYCLIsHost && !LangOpts.SYCLESIMDBuildHostCode)
+    if (LangOpts.SYCLIsDevice) {
```

Comment:
Why the additional braces? Formatting rules prefer consistency here. Either no braces for the `if `or we need braces for all the branches i.e. `else `as well.

Reply:
They are not needed, the legacy after a try to add a pass just after the if statement, will be removed.